src/modules/dashboard/components/ExpenditurePage/Split/SplitEqual/SplitEqual.tsx

```diff
@@ -0,0 +1,221 @@
+  const [, { value: recipients }] = useField<
+    { user?: AnyUser; amount?: number; key: string }[]
+  >('split.recipients');
+  const { tokens: colonyTokens } = colony || {};
+  const [, { value: amount }] = useField<{
+    value: string;
+    tokenAddress: string;
+  }>('split.amount');
+  const token = colony.tokens.find(
+    (colonyToken) => colonyToken.address === amount?.tokenAddress,
+  );
+  const { colonyAddress } = colony || {};
+  const { data: colonyMembers } = useMembersSubscription({
+    variables: { colonyAddress: colonyAddress || '' },
+  });
+
+  const selectedToken = useMemo(
+    () =>
+      colonyTokens.find(
+        (colonyToken) => colonyToken.address === amount.tokenAddress,
+      ),
+    [colonyTokens, amount.tokenAddress],
+  );
+
+  const recipientsCount = useMemo(() => {
+    const recip = recipients?.filter(
+      (recipient) => recipient?.user?.id !== undefined,
+    );
+
+    return recip.length || 0;
+  }, [recipients]);
+
+  const calculatedAmount = useMemo(() => {
+    return !amount.value ? 0 : Number(amount?.value) / (recipientsCount || 1);
+  }, [amount, recipientsCount]);
+
+  useEffect(() => {
+    setFieldValue(
+      'split.recipients',
+      recipients.map((recipient) => {
+        return { ...recipient, amount: calculatedAmount };
+      }),
+    );
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [calculatedAmount, setFieldValue]);
+
+  return (
+    <>
+      <FormSection appearance={{ border: 'bottom' }}>
+        <div className={styles.valueContainer}>
+          <div className={styles.inputContainer}>
+            <Input
+              name="split.amount.value"
+              appearance={{
+                theme: 'underlined',
+                size: 'small',
+              }}
+              label={MSG.amountLabel}
+              placeholder="Not set"
+              formattingOptions={{
+                numeral: true,
+                numeralDecimalScale: getTokenDecimalsWithFallback(
+                  selectedToken && selectedToken.decimals,
+                ),
+                numeralPositiveOnly: true,
+              }}
+              maxButtonParams={{
+                setFieldValue,
+                // mock, needs to be changed to the actual value
+                maxAmount: '100',
+                fieldName: 'split.amount.value',
+              }}
+            />
+          </div>
+          <div className={styles.tokenWrapper}>
+            <div>
+              <TokenSymbolSelector
+                label=""
+                tokens={colonyTokens}
+                name="split.amount.tokenAddress"
+                appearance={{ alignOptions: 'right', theme: 'grey' }}
+                elementOnly
+              />
+            </div>
+          </div>
+        </div>
+      </FormSection>
+      <FormSection appearance={{ border: 'bottom' }}>
+        <div className={styles.counterWrapper}>
+          <span>
+            {recipientsCount}{' '}
```

Comment:
Please, do this string via `FormattedMessage`, not good to have spaces in the html like that